src/Coypu.AcceptanceTests/WaitAndRetryExamples.cs

```diff
@@ -38,8 +38,8 @@ public void SetUp()
         protected void ApplyAsyncDelay()
         {
             // Hide the HTML then bring back after a short delay to test robustness
-            browser.ExecuteScript("window.holdIt = window.document.body.innerHTML;");
             browser.ExecuteScript("window.document.body.innerHTML = '';");
+            browser.ExecuteScript("window.holdIt = window.document.body.innerHTML;");
```

Comment:
I am verry sorry, I must have mixed those lines while trying to get the `FindAllCss` failing tests to pass. I executed the tests in both versions, they fail on both.

Consequently I reverted this change now. It indeed looks weird.

While investigating, I spotted that `fakefile.fsx` had a `TestAll` target defined which chained all other test argets together. I removed it in favor of independently executing test categories.